### PR TITLE
Move to use contra-env-infra templates

### PR DESCRIPTION
**Pull request by arilivigni**
* Update README with new examples and update current ones
  * Example without minishift and sample project
* Modify globals
  * versions of minishift/openshift
  * new helper_container variables
  * modify_scc to false
* Remove SAMPLE_PARAMS options
* Moved away from my fork since PR got merged
* Update TOC

### CODE REVIEW

File: playbooks/roles/os_temps/tasks/setup_os_templates.yml [hunk 1]
```diff
@@ -60,17 +60,6 @@
...
   when: build_config_name_check.stdout != ""
 
-- name: "{{ container_config_name }} :: Check if the template being loaded contains the parameter..."
-  shell: "{{ oc_bin }} describe template {{ template_name_file.stdout }} | grep {{ item.key }}"
-  register: check_temp_params
-  with_items: "{{ PARAMS }}"
-  ignore_errors: yes
-
-- set_fact:
-    params: "{{ params | default([]) }} + [ '-p {{ item.item.key }}={{ item.item.val }}' ]"
```
[the remark this answers is not in the document]
Comment (arilivigni) [reply playbooks/roles/os_temps/tasks/setup_os_templates.yml]: Made the change in recent commit please take a look and provide feedback

[the remark this answers is not in the document]
Comment (dirgim) [reply playbooks/roles/os_temps/tasks/setup_os_templates.yml]: Looks good now

File: README.md [hunk 2]
```diff
@@ -248,6 +264,130 @@ _Note: The -K is used to prompt you for your password for sudo (if you require o
...
+    2. OpenShift project -e openshift_project=ari-my-project 
+ 4. Modify my container tags with the default tag. tag=stable
+ 5. Don't setup or start minishift
+ 6. Don't run pre-reqs
+ 7. Dont setup nested virt
+
+```
+    ansible-playbook -vv -i "localhost," contra-env-setup/playbooks/setup.yml \
+    -e user=cloud-user -e project_repo=https://github.com/arilivigni/my-project-example \
+    -e openshift_project=ari-ci-pipeline -e setup_minishift=false -e start_minishift=true \
```
Comment (ifireball) [inline README.md]: whats the point of having `start_minishift=true` if `setup_minishift=false` ?

Comment (arilivigni) [reply README.md]: One starts minishift if it has already been setup but may be stopped and the other sets it up.  If you want more detail please look into the code.  This functionality has been there for a while and introduced by the CVP team.

**Pull request merged**